### PR TITLE
add IBM logo

### PR DESCRIPTION
**Pull request by metonym**

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit 77438957c3979b81334455e5cb0aa7ab93e65c1c

https://deploy-preview-403--svelte-community.netlify.app

**Review by dreitzner — APPROVED** [review]
nice

**Comment by dreitzner** [comment]
@sw-yx maybe add prominent?

File: whos-using-svelte/WhosUsingSvelte.js [hunk 1]
```diff
@@ -857,4 +857,9 @@ export const companies = [
 			src: "organisations/emakina-cee.svg",
 			alt: "Emakina CEE Logo"
 		},
+		{
+			href: "https://www.ibm.com/",
+			src: "organisations/ibm.svg",
+			alt: "IBM logo"
+		},
```
Comment (swyxio) [inline whos-using-svelte/WhosUsingSvelte.js]:
```suggestion
			prominent: true
		},
```

File: whos-using-svelte/WhosUsingSvelte.js [hunk 2]
```diff
@@ -857,4 +857,10 @@ export const companies = [
 			src: "organisations/emakina-cee.svg",
 			alt: "Emakina CEE Logo"
 		},
+		{
+			href: "https://www.ibm.com/",
+			src: "organisations/ibm.svg",
+			alt: "IBM logo"
```
Comment (swyxio) [inline whos-using-svelte/WhosUsingSvelte.js]:
```suggestion
			alt: "IBM logo",
```

**Comment by swyxio** [comment]
welcome IBM! @metonym would love more detail on where IBM uses Svelte currently?

**Comment by metonym** [comment]
@sw-yx Thanks! We currently use Svelte for internal prototypes and microsites.

Initially, designers used Svelte for rapid prototyping, not unlike [Daniel Sandoval's talk "Prototyping Design with Svelte"](https://youtu.be/vHHLLJA0b70?t=1040) at the 2020 Svelte Summit.

**Comment by swyxio** [comment]
thanks for sharing!

**Pull request merged**